requirements/anomaly.txt

```diff
@@ -2,3 +2,4 @@
 # Anomaly Requirements.
 anomalib==0.5.1
 kornia==0.6.9
+pytorch-lightning>=1.9.0,<1.10.0
```

Comment:
Isn't it more concise to use ~=1.9.0 or ==1.9.* ?

Reply:
done https://github.com/openvinotoolkit/training_extensions/pull/3145/commits/6dc05247c9488f154a2ecfd9c3556b11bbdc397c